flutter-gui-tests/testSrc/io/flutter/tests/gui/InspectorTest.kt

```diff
@@ -23,35 +30,32 @@ class InspectorTest : GuiTestCase() {
 
   @Test
   fun importSimpleProject() {
-    // TODO(messick) killall Simulator
     ProjectCreator.importProject()
     ideFrame {
-      selectSimulator()
-      Pause.pause(10000) // Give the simulator time to get started
       findRunApplicationButton().click()
-      runToolWindow.findContent("main.dart").waitForOutput(StringTextMatcher(".*Syncing files to device.*"), Timeouts.seconds10)
+      val runner = runToolWindow.findContent("main.dart")
+      pause(object : Condition("Start app") {
+        override fun test(): Boolean {
+          return runner.isExecutionInProgress
+        }
+      }, Timeouts.seconds10)
+      val inspector = flutterInspectorFixture()
+      inspector.activate()
+      pause(1000)
```

Comment:
Can we wait for a proper condition here to make this a little more robust?

Reply:
Yep! That was just to give me a glimpse of the screen before the IDE exited. Already changed locally.